index.html

```diff
@@ -48,8 +50,166 @@ <h3 class="heading-3">Let's connect</h3>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img class="dot"  src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang">javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+
+    <div class="work-section">
+    <img  class="cards" src="./images/snap-card-3.png" alt="availability">
+    <h2 class="heading-2">Multi-Post<br>Stories</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img  class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+    </div>
+
+    <div class="work-section">
+    <img class="cards" src="./images/Snap-card-5.png" alt="avocode">
+    <h2 class="heading-2">Tonic</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img class="dot"  src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img class="dot"  src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+
+    <div class="work-section">
+    <img class="cards" src="./images/snap-card-2.png" alt="avocode">
+    <h2 class="heading-2">Multi-Post<br>Stories</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img  class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang">html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+</div>
+  <!--- ===== About section ======= --->
+  <div class="about">
+    <h1 class="heading">About<br>Myself</h1>
+    <p class="paragraph">
```

Comment:
- [ ] The font-size of this `p` tag in the figma is `16px` but in your design its 1.9rem which is double than required. kindly make adjustments here and try to make it similar to the figma design. 
your design
![fd](https://user-images.githubusercontent.com/96683843/219850507-f2f0592f-3e0e-4129-9ced-1972931eadd7.jpg)
figma
![sd](https://user-images.githubusercontent.com/96683843/219850532-7b958944-f6b9-4f77-9ba1-e3af5262e4d5.jpg)